Client/Frontend/AuthenticationManager/PasscodeEntryViewController.swift

```diff
@@ -16,20 +16,32 @@ import SwiftKeychainWrapper
 /// Presented to the to user when asking for their passcode to validate entry into a part of the app.
 class PasscodeEntryViewController: BasePasscodeViewController {
     weak var delegate: PasscodeEntryDelegate?
-    fileprivate let passcodePane = PasscodePane()
-
+    fileprivate var passcodePane: PasscodePane?
```

Comment:
I'm not sure what the right thing to do here: I don't really like the idea of an optional here where none was there before it, but understand that under some circumstances we'd have to throw away a `PasscodePane`.

But hang on, on L23, you immediate set the value. In that case, please replace this with `var passcodePane: PasscodePane` (non-optional) and change the code accordingly.

Reply:
I think I changed this to an optional because the `passcodePane` value is set after `super.init` is called. This was because `authenticationInfo` is needed to initialize `passcodePane` and is setup in the super class. I'll look into restructuring it without the optional